README.md

```diff
@@ -8,6 +8,8 @@ Template to bootstrap and configure new projects maintained by the Flashbots col
 
 ## Instructions
 
+- Give a descriptive name to the repository. If it is a generic name like `docs`, call it `flashbots-docs` to make it clearer for people who fork it.
```

Comment:
Including 'flashbots' in every repository name seems redundant, as all repositories are already named under the Flashbots organization (this repository is 'http:/github.com/flashbots/flashbots-repository-template'). This is also would not apply retroactively to most of our repos, like mev-boost, mev-inspect, infra, etc, so it doesn't appear necessary to include this change. Maybe elaborating on what a discriptive name is, like including more words or defining acceptable specificity, but the 'flashbots' prefix seems to contribute to neither